src/Core/src/Handlers/SwipeView/SwipeItemMenuItemHandler.Windows.cs

```diff
@@ -0,0 +1,56 @@
+﻿using WSwipeItem = Microsoft.UI.Xaml.Controls.SwipeItem;
+
+namespace Microsoft.Maui.Handlers
+{
+	public partial class SwipeItemMenuItemHandler : ElementHandler<ISwipeItemMenuItem, WSwipeItem>
+	{
+		protected override WSwipeItem CreateNativeElement()
+		{
+			return new WSwipeItem();
+		}
+
+		public static void MapTextColor(SwipeItemMenuItemHandler handler, ITextStyle view)
+		{
+			var textColor = handler.VirtualView.GetTextColor();
```

Comment:
Thinking in devs extending, is not a priority but could be nice to move this to some extension methods.